common/src/main/java/com/tc/util/version/VersionCompatibility.java

```diff
@@ -47,6 +47,20 @@ private static boolean isCompatible(Version v1, Version v2) {
     return ((v1.major() == v2.major()) && (v1.minor() == v2.minor()));
   }
 
+  public static boolean isNewer(Version v1, Version v2) {
+    if (v1 == null || v2 == null) { throw new NullPointerException(); }
+    if (v1.major() > v2.major()) {
```

Comment:
`return v1.compareTo(v2) > 0`

Reply:
that goes very deep including patch and build number.  I don't think we want to go that deep with this.